client/js/screens/Browse/Browse.js

```diff
@@ -1,10 +1,61 @@
 import React from 'react';
-import {View, Text} from 'react-native';
+import {View, Text, Image, ScrollView, TouchableOpacity} from 'react-native';
+import styles from './styles';
 
-const Browse = () => {
+const Browse = ({navigation}) => {
   return (
     <View>
```

Comment:
No need to wrap your scroll view with the view component. Please remove View component.